gatsby-config.js

```diff
@@ -646,154 +382,6 @@ module.exports = {
                     }
```

Comment:
For Target we can remove event reference. And we can keep Deprecated APIs here and maybe remove from AEP repo, @swarna04 what do you think?